pacu/main.py

```diff
@@ -685,9 +685,13 @@ def parse_data_command(self, command: List[str]) -> None:
             self.print('\nSession data:')
             session.print_all_data_in_session()
         else:
-            service = command[1].upper()
+            service = command[1]
+            if service.upper() in session.aws_data_field_names:
+                service = service.upper()
+
             if service not in session.aws_data_field_names:
```

Comment:
Hmm, I'm still having issues here. Does this work for you?

```suggestion
            if service.upper() not in [svc.upper() for svc in session.aws_data_field_names]:
```

Reply:
service like `CodeBuild` must use case sensitive some aren't. Like`IAM` or `EC2` both works fine with case insensitive.
```
APIGateway      CloudTrail      CloudWatch      CodeBuild       Config  DataPipeline    
DynamoDB        EC2     ECS     Glue    GuardDuty       IAM     Inspector       Lambda  
Lightsail       S3      SecretsManager   Shield  SSM     VPC     WAF     Account 
AccountSpend    Route53 RDS
```

Reply:
```python
if service.upper() not in [svc.upper() for svc in session.aws_data_field_names]:
```
`codebuild` will pass the check but will fail in `getattr(session, service)` cause did't have  `codebuild` service name in `session.aws_data_field_names` only have service name `CodeBuild`

Reply:
Yeah so was hoping to work around that by converting everything to uppercase, looks like we need to do it for retrieving the data as well. Will make another suggestion here.